docs/index.rst

```diff
@@ -7,13 +7,13 @@ Presto-Admin
 
 Introduction
 ------------
-Presto-Admin is a tool for installing and managing the Presto query engine on a
-cluster. It provides easy-to-use commands to:
+Presto-Admin is a tool for installing and managing the Presto query engine
```

Comment:
Dash or no dash? see title?

Reply:
I standardized on init cap and no hyphen for the product name: Presto Admin

and lowercase with hyphen for the command name: presto-admin

I must have missed that one instance.

Reply:
Fixed with a single-character push.

Reply:
Excellent!